app/renderer/js/utils/domain-util.js

```diff
@@ -20,7 +20,10 @@ const logger = new Logger({
 
 let instance = null;
 
-const defaultIconUrl = '../renderer/img/icon.png';
+// __dirname returns the path to the directory similar to: '/home/user/zulip-electron/app/renderer/js/utils'
+// Slicing it from 0 to -9 gives us just '/home/user/zulip-electron/app/renderer'
+// To this, we concatinate '/img/icon.png' with which now one can refer to the fallback image from anywhere in the repository
+const defaultIconUrl = __dirname.slice(0, -9) + '/img/icon.png';
```

Comment:
It does feel like a hack but using `__dirname` and slicing it, we get the whole path and one that is Independent of where you are accessing it from. You can refer to the `defaultIconUrl` from anywhere in the code base and it points to the same image. I thought this way would make it bullet proof for future in case this path is referred at any place that isn't coved by declaring it as just ../../img/icon.png which completely depends on the `pwd`.
Again, this is what looked good to me but if you have a better approach i mind, I'll love to hear it and would be glad to implement it right away (^_^)

Reply:
Yeah, second approach is preferable according to me.